### DISCUSSION TITLE
winget ?

### DISCUSSION THREAD

**Discussion by wincmd64** [ALTaleX531/OpenGlass#54] [Q&A] [answered]
I can't find the OpenGlass in winget

**Comment by ALTaleX531** [comment, marked as answer]
because it is not included in the winget.

**Reply by wincmd64** [reply]
![p](https://github.com/user-attachments/assets/2438ea35-9b01-4c37-8867-cf5b1f5bb1d1)

> because it is not included in the winget.

so manually downloading an archive, unpacking it and running the batch file is better than pressing one button?

**Comment by wincmd64** [comment]
why?

**Reply by ALTaleX531** [reply]
![image](https://github.com/user-attachments/assets/21264073-51ba-4907-9d48-aced653371b6)
![image](https://github.com/user-attachments/assets/f39f95a5-de5e-4b1a-b091-c141584c4443)

just stop asking😑

**Comment by wincmd64** [comment]
If you find adding archives to winget be challenging, Windows is not for you. 🤣🤣

**Reply by Sand216** [reply]
I deleted that comment almost instantly as it was just a joke, but since you acknowledged it I’ll put it back for reference:

> Not all programs can be magically added to winget. If you find extracting archives to be challenging, OpenGlass is not for you.

Didn’t mean to escalate this further with a joke comment, sorry ALTaleX.

**Comment by ALTaleX531** [comment]
Apparently you don't realize how annoying your behavior is, nor do you understand that developers are not obligated to add things to winget, and that your attitude in requesting certain things is even completely insincere.🥱

If you find requesting something in a sincere manner challenging, then github is not for you.👎
